src/ast/mod.rs

```diff
@@ -3441,6 +3441,37 @@ pub enum Statement {
     ///
     /// See <https://learn.microsoft.com/en-us/sql/t-sql/statements/set-statements-transact-sql>
     SetSessionParam(SetSessionParamKind),
+    /// RaiseError (MSSQL)
```

Comment:
Can we add a link in the doc to the syntax? that would make it easier to find when folks come across the variant.

Also the identation of the variant seems off, the PR might need a `cargo fmt` run?